extensions/debank/src/index.tsx

```diff
@@ -0,0 +1,322 @@
+            {favoriteAddresses.map((address) => (
+              <List.Item
+                key={address.address}
+                title={address.identifier || address.address}
+                accessoryTitle={!address.identifier ? "" : address.address}
+                icon={{
+                  source: Icon.Star,
+                  tintColor: Color.Yellow,
+                }}
+                actions={
+                  <ActionPanel>
+                    <ActionPanelItem
+                      title={`Show Balance`}
+                      icon={Icon.Binoculars}
+                      onAction={() => {
+                        push(<BalanceView address={address.address} />);
+                      }}
+                    />
+                    <OpenInBrowserAction url={`https://debank.com/profile/${address.address}`} title="Open on DeBank" />
+                    <ActionPanelItem
+                      title={`Remove from Favorites`}
+                      icon={Icon.Star}
+                      shortcut={{ modifiers: ["cmd", "shift"], key: "f" }}
+                      onAction={async () => {
+                        confirmAlert({
+                          title: "Confirm Removal",
+                          message: `Are you sure you want to remove "${
+                            address.identifier || address.address
+                          }" from your favorites?`,
+                          primaryAction: {
+                            title: "Remove",
+                            style: AlertActionStyle.Destructive,
+                            onAction: () => removeFavoriteAddress(address.address),
+                          },
+                        });
+                      }}
+                    />
+                  </ActionPanel>
+                }
+              />
+            ))}
+          </ListSection>
+
+          <ListSection title="Search for Address">{defaultItem}</ListSection>
+        </>
+      )}
+    </List>
+  );
+}
+
+type AddFavoriteAddressFormProps = {
+  address: string;
+  onSubmit: (values: any) => void;
+};
+
+function AddFavoriteAddressForm(props: AddFavoriteAddressFormProps) {
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <SubmitFormAction title="Add to Favorites" onSubmit={props.onSubmit} />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField title="Address" id="address" defaultValue={props.address} />
+      <Form.TextField title="Identifier" id="identifier" />
+    </Form>
+  );
+}
+
+type BalanceViewProps = {
+  address: string;
+};
+
+export function BalanceView(props: BalanceViewProps) {
+  const [balance, setBalance] = useState<TotalBalance | null>(null);
+
+  useEffect(() => {
+    if (props.address) {
+      getTotalBalance(props.address).then((data) => {
+        setBalance(data);
+      });
+    }
+  }, []);
+
+  const { push } = useNavigation();
+
+  if (!balance) {
+    return (
+      <List>
+        <List.Item key="1" title="Loading..." />
```

Comment:
the recommended UX when loading a list content is to render an empty List with `isLoading` set to true (https://developers.raycast.com/api-reference/user-interface/list#list)